saviSimulation/savi/jason_processing/FieldAntenna.java

```diff
@@ -1,28 +1,103 @@
 package savi.jason_processing;
 
+import java.awt.*;
+import java.awt.event.*;
 import java.util.LinkedList;
 import java.util.List;
 
+import javax.swing.JButton;
```

Comment:
I had problems running the model but they are fixed. I needed to change the java compiler version to 1.8 (my configuration was with 1.3). Maybe is something to remark in the wiki